src/dataExplorer/components/SaveAsTaskForm.tsx

```diff
@@ -136,7 +137,7 @@ const mstp = (state: AppState) => {
   const activeQuery = getActiveQuery(state)
   const org = getOrg(state)
   const userDefinedVars = getAllVariables(state)
-    .map(v => asAssignment(v))
+    .map(v => asAssignmentNode(v))
```

Comment:
A bit of explicit naming updates.
`Variable` == UI abstraction/type
`VariableAssignment` == flux AST for a Variable